web/src/app/alerts/CreateAlertDialog/StepContent/index.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import gql from 'graphql-tag'
+import { useQuery } from 'react-apollo'
+import _ from 'lodash-es'
+import Step0 from './Step0'
+import Step1 from './Step1'
+import Step2 from './Step2'
+import Step3 from './Step3'
+
+const query = gql`
+  query($input: ServiceSearchOptions) {
+    services(input: $input) {
+      nodes {
+        id
+        name
+        isFavorite
+      }
+    }
+  }
+`
+
+export default props => {
```

Comment:
Let's put this in `StepContent.js` and we can export it from `index.js`. That way we keep filenames matching the components they contain so things are easy to search for.